src/display/api.js

```diff
@@ -1088,16 +1088,24 @@ var PDFWorker = (function PDFWorkerClosure() {
   var nextFakeWorkerId = 0;
 
   function getWorkerSrc() {
+    var suffix;
```

Comment:
I don't see why these two have to be at the top of the function. Let's just declare them where they are actually created and used, i.e., at lines 1102 and 1103.